simple-calculator/mycalculator.ts

```diff
@@ -0,0 +1,33 @@
+function calculator(x:number) {
```

Comment:
fix the code style please spacing and stuff, either check Brahim's code or the styleguide.